packages/common-resources/static-resources/src/i18n/zh_CN/common.ts

```diff
@@ -177,7 +178,7 @@ export default {
     labelConnectBy: '当前连接 <1>{{connectBy}}</1>',
     labelWrongNetwork: '未识别网络',
     labelActivatedAccountDeposit: '首次开通二层钱包需充值并激活',
-    labelActivatedAccountNotSupport: 'We are apologize, Loopring app not support your address current now.',
+    labelActivatedAccountNotSupport: 'We apologize, loopring app does not support your address currently.',
```

Comment:
This is technically fine, but a more native sentence would probably sound like "Loopring does not currently support your address. We apologize for the inconvenience."